samples/chat-app/server/Program.fs

```diff
@@ -29,7 +29,6 @@ let main args =
     if app.Environment.IsDevelopment () then
         app.UseGraphQLAltair "/altair" |> ignore
         app.UseGraphQLGraphiQL "/graphiql" |> ignore
-        app.UseGraphQLPlayground "/playground" |> ignore
```

Comment:
The package is not deprecated, maybe just a method is deprecated?

Reply:
Now that you pointed that out, I just saw [that the whole package `GraphQL.Server.Ui.Playground` is deprecated](https://github.com/graphql-dotnet/server). Using GraphQLPlayground here was [causing the builds to fail](https://github.com/fsprojects/FSharp.Data.GraphQL/actions/runs/12153973380).

Reply:
Ah, I see. The benefit of playground is the ability to get schema in DSL format.
If it is deprecated, then let's remove it

Reply:
Then we also need to remove a line in `Packages.props`